portal/styles/button.scss

```diff
@@ -4,4 +4,8 @@
   text-decoration: underline;
   cursor: pointer;
   color: $govuk-link-colour;
+  padding: 0;
```

Comment:
In the introduction you mentioned
> Noticed that in the other instance where `button-as-link` was used, the `gov-uk-link` class was also used. Decided to include the additional styling from `gov-uk-link` that wasn't already in `button-as-link` in the latter class, for clarity and so that the developer doesn't have to add multiple classes to ensure that a button is styled as a GOV UK link.

I'm a bit wary of this because what if the `gov-uk-link` class changes? Do we have any way of using Sass to make `button-as-link` extend the `gov-uk-link` class?

If we can't find a way to keep it in sync automatically then we might be better off having to add both classes every time

Reply:
Great shout, have used the SASS @extend syntax